docs/modbus-scalability-test.md

```diff
@@ -0,0 +1,184 @@
+# Modbus Scalability Test
+Modbus Scalability Test is used to measure the EdgeX's ability when the device number scale-up.
+
+## Test Environment Overview
+![image](./images/modbus-scalability-test-overview.png)
+
+Since the test needs to handle a lot of ports for Modbus simulators, so we consider using the host network with docker https://docs.docker.com/network/host/, and the host networking driver only works on Linux hosts.
+
+Required tool:
+- docker
+- docker-compose
+
+## Prepare Test Environment
+
+### 1. Deploy southbound Modbus simulator
+Run the following command to deploy simulators:
+```
+docker run --rm -d --network host --name modbus-simulator \
+    -e SIMULATOR_NUMBER=1000 -e STARTING_PORT=10000 \
+    nexus3.edgexfoundry.org:10003/edgex-devops/edgex-modbus-simulator
```

Comment:
Could you please add an annotation for the arm64 image?

Reply:
Added.